app/controllers/browse_controller.rb

```diff
@@ -1,6 +1,4 @@
 class BrowseController < ApplicationController
-  enable_request_formats show: [:json]
```

Comment:
Why were we supporting json formats before, but not now. Not sure I understand how that relates to miller columns.

Reply:
The JSON was consumed by the Javascript to build up miller columns.

Reply:
Ah gotcha, thank you